core/src/test2/README.md

```diff
@@ -0,0 +1,3 @@
+Tests without any external dependencies.
```

Comment:
`test2`?  :). Maybe something like `independenttests` instead, so the meaning is slightly clearer?